Gemfile

```diff
@@ -4,11 +4,12 @@ source 'https://rubygems.org'
 
 gemspec
 
-platform :rbx do
-  gem 'rubysl-bigdecimal', '~> 2.0.2'
-  gem 'rubysl-coverage',   '~> 2.0.3'
-end
-
 group :development, :test do
-  gem 'devtools', git: 'https://github.com/rom-rb/devtools.git'
+  gem 'rspec',        '~> 3.8', '>= 3.8.0'
+  gem 'mutant',       github: 'mbj/mutant'
```

Comment:
@dkubb Mutant is very stable now, yo ushould not be in the need to pin a specific github commit anymore.